ethereum_bridge/src/protocol/transactions/votes/storage.rs

```diff
@@ -36,21 +38,39 @@ where
     Ok(())
 }
 
+/// Delete a tally from storage, and return the associated payload of
+/// type `T` being voted on, in case it has accumulated more than 1/3
+/// of fractional voting power behind it.
+#[must_use = "The optional payload returned by this function must be used"]
```

Comment:
nit: the word "payload" seems less natural than something like "value" since this is coming from storage.

Reply:
sure, can change this